<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_IP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Broker network address</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SubTop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Subscribe topic</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PubTop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Publishing topic</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>cID</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Uname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Pwd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="serverStartButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_programTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>36</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>261</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>true</italic>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color: rgba(0, 0, 0, 0);}</string>
    </property>
    <property name="text">
     <string>Server Configuration</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_IPEditor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PortEditor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SubTopEditor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PubTopEditor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cIDEditor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_UnameEditor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PwdEditor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
